content/en/docs/Tutorials/basic.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Basic: build a simple pipeline"
+weight: 410
+date: 2020-11-13
+description: >
+    A step-by-step tutorial to learn how to build a Baker pipeline using the included components
+---
+In this tutorial you'll learn how to create a Baker-based program to process a given dataset
+(in CSV format), filter records based on your needs and save the result to a file.
+
+The dataset we're going to use is an open dataset containing ratings on many
+[Ramens](https://www.kaggle.com/residentmario/ramen-ratings).
```

Comment:
Since we're jumping from data processing to food, a small sentence might help for the transition here. Plus I'm not sure everybody know what a ramen is
```suggestion
The dataset we're going to use is an open dataset containing ratings on many 
[Ramens](https://www.kaggle.com/residentmario/ramen-ratings), the famous japanese noodle soop!
```

or something like that maybe

Reply:
> I'm not sure everybody know what a ramen is

Shame on them! :laughing: 

fixing..